Resources/Prototypes/GameRules/events.yml

```diff
@@ -175,8 +178,7 @@
       path: /Audio/Announcements/attention.ogg
     startDelay: 10
     earliestStart: 30
-    minimumPlayers: 35
```

Comment:
I'd be careful removing this from MouseMigration since it also includes ratking spawns, which can be pretty fucking devastating to smaller crews. VentCritters is _just_ mice. (Which also has a 15 min pop, what?)

Reply:
Glad someone reads my PRs.